ofctrl/ofSwitch.go

```diff
@@ -108,7 +110,9 @@ func NewSwitch(stream *util.MessageStream, dpid net.HardwareAddr, app AppInterfa
 	}
 	// Prepare a context for current connection.
 	s.ctx, s.cancel = context.WithCancel(context.Background())
-	s.tlvMgr = newTLVMapMgr()
+	if app.TLVMapEnabledOnSwitch() {
```

Comment:
Since the return value is always true, why add an if here?

Reply:
The `ofActor` in the test is an implementation of the Interface `AppInterface`. Returning true in the test code is to ensure the related code branch is able to tested. For other implementation like antrea, the return value may be false.